### PR TITLE
autotools: fix parallel build in ofxconnect/Makefile.am

### LINKED ISSUE #76: missing version variables
I just installed 0.10.6 on Gentoo, and then  tried to reinstall KMyMoney, which failed with 
In file included from /usr/include/qt5/QtCore/qiterator.h:43,
                 from /usr/include/qt5/QtCore/qmap.h:43,
                 from /usr/include/qt5/QtCore/QMap:1,
                 from /var/tmp/portage/app-office/kmymoney-9999/work/kmymoney-9999/kmymoney/interfaces/kmymoneyplugin.h:16,
                 from /var/tmp/portage/app-office/kmymoney-9999/work/kmymoney-9999/kmymoney/plugins/ofx/import/ofximporter.h:22,
                 from /var/tmp/portage/app-office/kmymoney-9999/work/kmymoney-9999/kmymoney/plugins/ofx/import/ofximporter.cpp:9:
/var/tmp/portage/app-office/kmymoney-9999/work/kmymoney-9999/kmymoney/plugins/ofx/import/ofximporter.cpp:381:5: error: operator '<<' has no left operand
  381 | #if QT_VERSION_CHECK(LIBOFX_MAJOR_VERSION, LIBOFX_MINOR_VERSION, LIBOFX_MICRO_VERSION) >= QT_VERSION_CHECK(0, 10, 0)

Poking into various files, I find the following in the libofx.h installed by 0.10.6

#define LIBOFX_MAJOR_VERSION 
#define LIBOFX_MINOR_VERSION 
#define LIBOFX_MICRO_VERSION 
#define LIBOFX_BUILD_VERSION 0

Shouldn't those contain the actual values?  (0, 10, and 6)  What I can't figure out is whether this is an issue in libofx itself, or in the Gentoo ebuild, so sorry if I've filed this in the wrong place.

### PR DESCRIPTION
**Pull request by thesamesam**
Closes: https://github.com/libofx/libofx/issues/76
Signed-off-by: Sam James <sam@gentoo.org>

### CODE REVIEW

File: ofxconnect/Makefile.am [hunk 1]
```diff
@@ -23,9 +23,9 @@ EXTRA_DIST = cmdline.ggo test-privateserver.sh CMakeLists.txt
 
 # See README.privateserver for details on this server and how to get
 # the key needed to run this test.
-TESTS = test-privateserver.sh 
+TESTS = test-privateserver.sh
 
-ofxconnect.1: $(top_srcdir)/configure.ac
```
Comment (cstim) [inline ofxconnect/Makefile.am:28]: I understand from your commit message that the missing dependency to `ofxconnect` breaks the parallel build (which is obviously something I didn't test here). 

However, introducing the dependency again will then break the target `make distcheck`, which is why I just removed it in 76dae192 last weekend. If you introduce this dependency, then run `make distcheck`, it eventually fails with

```
ERROR: files left in build directory after distclean:
./ofxdump/ofxdump.1
./ofxconnect/ofxconnect.1
make[1]: *** [Makefile:826: distcleancheck] Fehler 1
```
From https://www.gnu.org/software/automake/manual/html_node/Errors-with-distclean.html I got the impression the error at distcheck is avoided by dropping this dependency. If you can suggest a different change, feel free to adapt your PR.

File: configure.ac [hunk 2]
```diff
@@ -57,9 +55,6 @@ CXXFLAGS="-DIN_LIBOFX $CXXFLAGS"
 
 LIBOFX_VERSION=$LIBOFX_MAJOR_VERSION.$LIBOFX_MINOR_VERSION.$LIBOFX_MICRO_VERSION
 
-AC_SUBST(LIBOFX_MAJOR_VERSION)
-AC_SUBST(LIBOFX_MINOR_VERSION)
-AC_SUBST(LIBOFX_MICRO_VERSION)
```
Comment (cstim) [inline configure.ac:62]: The changes here are totally understandable. I'll cherry-pick them already.

**Review by cstim — CHANGES_REQUESTED** [review]
I approve the part in configure.ac. I'll cherry-pick that commit for now already.

I have some doubts about the ones in Makefile.am, as explained there.

File: configure.ac (hunk 2 above)
Comment (thesamesam) [reply configure.ac:62]: Thanks!

File: ofxconnect/Makefile.am (hunk 1 above)
Comment (thesamesam) [reply ofxconnect/Makefile.am:28]: Ah, right. Let me take a look.

**Pull request closed without merging**